data/Syria/Majlis/sources/archive/term-2012-images.csv

```diff
@@ -0,0 +1,275 @@
+275,http://parliament.gov.sy/SD08/msf/1430118499_.png
+276,http://parliament.gov.sy/SD08/msf/1421665028_.jpg
+277,http://parliament.gov.sy/SD08/msf/1430118318_.png
+278,http://parliament.gov.sy/SD08/msf/1430118240_.png
+279,http://parliament.gov.sy/SD08/msf/1430049155_.png
+280,http://parliament.gov.sy/SD08/msf/1430047929_.png
+281,http://parliament.gov.sy/SD08/msf/1430043357_.png
+282,http://parliament.gov.sy/SD08/msf/1430042653_.png
+283,http://parliament.gov.sy/SD08/msf/1430042532_.png
+284,http://parliament.gov.sy/SD08/msf/1430042388_.png
+285,http://parliament.gov.sy/SD08/msf/1430042274_.png
+286,http://parliament.gov.sy/SD08/msf/1430042188_.png
+287,http://parliament.gov.sy/SD08/msf/1430042082_.png
+288,http://parliament.gov.sy/SD08/msf/1430041994_.png
+289,http://parliament.gov.sy/SD08/msf/1430041897_.png
+290,http://parliament.gov.sy/SD08/msf/1430041816_.png
+291,http://parliament.gov.sy/SD08/msf/1430041549_.png
+292,http://parliament.gov.sy/SD08/msf/1430041441_.png
+293,http://parliament.gov.sy/SD08/msf/1430041244_.png
+294,http://parliament.gov.sy/SD08/msf/1430041145_.png
+295,http://parliament.gov.sy/SD08/msf/1430041055_.png
+296,http://parliament.gov.sy/SD08/msf/1430040926_.png
+297,http://parliament.gov.sy/SD08/msf/1430040761_.png
+298,http://parliament.gov.sy/SD08/msf/1430040487_.png
+299,http://parliament.gov.sy/SD08/msf/1430039751_.png
+300,http://parliament.gov.sy/SD08/msf/1430038932_.png
+301,http://parliament.gov.sy/SD08/msf/1430038099_.png
+302,http://parliament.gov.sy/SD08/msf/1430037668_.png
+303,http://parliament.gov.sy/SD08/msf/1421746465_.jpg
+304,http://parliament.gov.sy/SD08/msf/1430036414_.png
+305,http://parliament.gov.sy/SD08/msf/1430036320_.png
+306,http://parliament.gov.sy/SD08/msf/1430035934_.png
+307,http://parliament.gov.sy/SD08/msf/1430035658_.png
+308,http://parliament.gov.sy/SD08/msf/1430035198_.png
+309,http://parliament.gov.sy/SD08/msf/1431505111_.png
+310,http://parliament.gov.sy/SD08/msf/1430034932_.png
+311,http://parliament.gov.sy/SD08/msf/1430034837_.png
+312,http://parliament.gov.sy/SD08/msf/1430034515_.png
+313,http://parliament.gov.sy/SD08/msf/1421749315_.jpg
+314,http://parliament.gov.sy/SD08/msf/1421753138_.jpg
+315,http://parliament.gov.sy/SD08/msf/1430034076_.png
+316,http://parliament.gov.sy/SD08/msf/1429790078_.png
+317,http://parliament.gov.sy/SD08/msf/1429789826_.png
+318,http://parliament.gov.sy/SD08/msf/1429789691_.png
+319,http://parliament.gov.sy/SD08/msf/1429789577_.png
+320,http://parliament.gov.sy/SD08/msf/1429789420_.png
+321,http://parliament.gov.sy/SD08/msf/1429789178_.png
+322,http://parliament.gov.sy/SD08/msf/1429789009_.png
+323,http://parliament.gov.sy/SD08/msf/1429788914_.png
+324,http://parliament.gov.sy/SD08/msf/1421754634_.jpg
+325,http://parliament.gov.sy/SD08/msf/1421754919_.jpg
+326,http://parliament.gov.sy/SD08/msf/1421755207_.jpg
+327,http://parliament.gov.sy/SD08/msf/1429701135_.png
+330,
+331,
+332,http://parliament.gov.sy/SD08/msf/1434363648_.png
+363,
+364,
```

Comment:
It's a little odd to include all these blank lines here — that's essentially saying that there's explicitly no image for these people, rather than we just don't know of one.